policyengine_us/variables/gov/irs/income/taxable_income/deductions/itemizing/itemized_taxable_income_deductions_reduction.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class itemized_taxable_income_deductions_reduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Itemized taxable income deductions reduction"
+    unit = USD
+    definition_period = YEAR
+
```

Comment:
This works right?
```suggestion
    defined_for = "gov.irs.deductions.itemized.reduction.applies"

```

Reply:
I dont think so, it is looking for a variable

Reply:
OK filed https://github.com/PolicyEngine/policyengine-core/issues/341